### ISSUE TITLE
Running the example as a package

### ISSUE THREAD

**Issue by annayqho** [annayqho/TheCannon#15]
Should be able to download some package and run the example (so that the directory structure is all correct, etc)

**Comment by davidwhogg** [comment]
i agree!  And probably download should be

    pip install TheCannon

**Comment by annayqho** [comment]
OK, the labels file and data files needed to run the code (for a training set and test set of the 523 cluster stars) are in example_DR10.tar.gz, which is in the TheCannon/examples folder. The instructions for running the code will be to simply unpack this file inside the main TheCannon directory. 

And it can now be installed as "pip install TheCannon". Running the following works on my computer:
pip install TheCannon
python setup.py install --user
(and then from any directory):
import TheCannon

and one should be able to run it by doing "from TheCannon import run_apogee_example", after I fix the current bugs in the rest of the code...
